cmake/FDBComponents.cmake

```diff
@@ -163,6 +163,9 @@ endif()
 
 set(SSD_ROCKSDB_EXPERIMENTAL ON CACHE BOOL "Build with experimental RocksDB support")
 set(PORTABLE_ROCKSDB ON CACHE BOOL "Compile RocksDB in portable mode") # Set this to OFF to compile RocksDB with `-march=native`
+set(ROCKSDB_SSE42 OFF CACHE BOOL "Compile RocksDB with SSE42 enabled")
```

Comment:
Just curios why it is OFF?

Reply:
I default to`ROCKSDB_SSE42=OFF` because I do not want to change the current (implicit) build configuration. This wayt no one needs to know about this option unless they want it enabled. (we can change the default once this option proves itself) and `USE_AVX` defaults to `ON`. In the event that someone wants to set it to `OFF` this will make sure that it is off for RocksDB as well.